Sources/BraveNews/Composer/FeedDataSource.swift

```diff
@@ -403,7 +401,7 @@ public class FeedDataSource {
     var varianceBySource: [String: Double] = [:]
     return rssItems.map {
       var content = $0
-      let recency = log(max(1, -content.publishTime.timeIntervalSinceNow))
+      let recency = Darwin.log(max(1, -content.publishTime.timeIntervalSinceNow))
```

Comment:
Why are you using `Darwin.log` here?

Reply:
No changes here, in first iteration of this PR there was a name conflict, `log()` was a logging function and colluded with this method so i had to specify module.
Now logs are called via `Logger.module.debug("xx")` so I can revert this change

Reply:
I see, thanks. I thought it was a macOS-related logger for some reason